packages/components/organisms/f-offers/src/components/ModalCopiedVoucherCode.vue

```diff
@@ -5,31 +5,32 @@
         is-narrow
         :data-test-id="testIds.modal"
         @close="close">
-        <h3>{{ $t('offersInbox.voucherCodeModal.title') }}</h3>
-        <p v-html="$t('offersInbox.voucherCodeModal.body')" />
-        <a
+        <h3>{{ $t('authenticated.voucherCodeModal.title') }}</h3>
+        <p v-html="$t('authenticated.voucherCodeModal.body')" />
+        <f-button
             v-if="hasLink"
             :href="url"
             :data-test-id="testIds.confirm"
-            class="o-btn o-btn--primary o-btn--block o-btn--rounded">
-            {{ $t('offersInbox.voucherCodeModal.confirmButton') }}
-        </a>
-        <button
+            :is-full-width="true">
```

Comment:
Can use a shorthand here:

```suggestion
            is-full-width>
```

Reply:
No longer added so resolving